### PR TITLE
2019 Hyundai Santa Fe fixes

### PR DESCRIPTION
**Pull request by cdwillie76**

### CODE REVIEW

**Comment by cdwillie76** [comment]
Is there anything else I can provide to help move this forward?  I don't see anything in the failed checks I can address.

**Comment by adeebshihadeh** [comment]
Can you provide a stock route? You can toggle between openpilot/stock in the settings.

**Comment by cdwillie76** [comment]
Here is a route: https://my.comma.ai/useradmin/?onebox=3c7ce223e3571b54%7C2020-11-22--20-18-45

File: selfdrive/car/hyundai/carcontroller.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, visual_alert,
         self.last_resume_frame = frame
 
     # 20 Hz LFA MFA message
-    if frame % 5 == 0 and self.car_fingerprint in [CAR.SONATA, CAR.PALISADE, CAR.IONIQ, CAR.KIA_NIRO_EV]:
+    if frame % 5 == 0 and self.car_fingerprint in [CAR.SONATA, CAR.PALISADE, CAR.IONIQ, CAR.KIA_NIRO_EV, CAR.SANTA_FE]:
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/carcontroller.py:83]: This message isn't in the stock route. why was this added?
